app/addr_prefixes.go

```diff
@@ -1,8 +1,9 @@
-package types
```

Comment:
Do we need to change the address prefixes as well from `xrn` to `regen`?

Reply:
I think yes. Let's check with @aaronc .

Reply:
What do people prefer @clevinson @amaurymartiny @blushi @anilCSE ?

Reply:
`regen` looks better and consistent